playground/Qdrant/Qdrant.AppHost/Program.cs

```diff
@@ -3,7 +3,7 @@
 
 var builder = DistributedApplication.CreateBuilder(args);
 
-var qdrant = builder.AddQdrant("qdrant")
+var qdrant = builder.AddQdrant("qdrant", httpPort:5503)
```

Comment:
```suggestion
var qdrant = builder.AddQdrant("qdrant")
```

remove

Reply:
Was in there to have a validated manifest test...you don't see the need?